### PR TITLE
Add support of importing imaginary literals

### PR DESCRIPTION
**Pull request by martong**

### CODE REVIEW

File: unittests/AST/ASTImporterTest.cpp [hunk 1]
```diff
@@ -519,6 +519,14 @@ TEST_P(ImportExpr, ImportFloatingLiteralExpr) {
           floatLiteral(equals(1.0e-5f), hasType(asString("float"))))));
 }
 
+TEST_P(ImportExpr, ImportImaginaryLiteralExpr) {
+  MatchVerifier<Decl> Verifier;
+  testImport(
+      "void declToImport() { (void)1.0i; }",
+      Lang_C, "", Lang_C, Verifier,
```
Comment (balazske) [inline unittests/AST/ASTImporterTest.cpp]: Lang_CXX14 should be used?

Comment (martong) [reply unittests/AST/ASTImporterTest.cpp]: Ok, changed it.

**Comment by martong** [comment]
Fixes #455

**Review by balazske — APPROVED** [review]
Looks good, but does it fix itself ?

**Comment by martong** [comment]
Yes, it does fix this:
```
Assertions:
      1 `false && "Unsupported Expr at import"' failed.
```

**Comment by martong** [comment]
https://reviews.llvm.org/D50428

**Pull request merged**